### DISCUSSION TITLE
mspider/user-guide/security/peer/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1293] [Announcements]
# mspider/user-guide/security/peer/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/security/peer/

**Comment by liyuerich** [comment]
所有图片中最右侧的操作列的 “操作” 2个字现在都没有了
